.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
This is not needed. You can add to your local .gitignore if you want to always ignore that.